sklearn/metrics/ranking.py

```diff
@@ -383,6 +386,12 @@ def precision_recall_curve(y_true, probas_pred, pos_label=None,
         Increasing thresholds on the decision function used to compute
         precision and recall.
 
+    References
+    ----------
+    .. [1] Manning, C. D., Raghavan, P., & Schütze, H. (2008). 
```

Comment:
I don't know where he's from but if he's German you can write Schuetze. I don't think anyone would kill you for Schutze, though.